### ISSUE TITLE
Health facilities vs Laboratories

### ISSUE THREAD

**Issue by RafaelKluender** [hzi-braunschweig/SORMAS-Project#2091]
Currently a facility can only be a health facility OR a Laboratory. 
However, often hospitals have laboratories included. 
I have imported a list of hospitals and now I discovered that there are no Laboratories. 
While trying to import the same facilities as a "laboratory" I ran into duplicate errors. 

Only workaround is to import those facilities again in a mode 
"Laboratory of Hospital X" but then we get two entries for the same facility.

**Comment by bernardsilenou** [comment]
@RafaelKluender Normally, hospital and labs are independent in the system even if they belong to the same physical facility in real life. 
It should be possible to have the same name, one as lab and another as hospitals.
Duplicate error should normally not occur. Please can you give more information?
 @MateStrysewskeSym is this normal?

**Comment by MateStrysewskeSym** [comment]
Yes, because both share the same Facility entity in the backend. We'll have a look into this and see whether we can change the duplicate detection.

**Comment by RafaelKluender** [comment]
I suggest to add a column in the facility import csv file: 
Laboratory included (TRUE/FALSE) and make the import feature to include automatically a laboratory with the same name or with a prefix like "Lab_Hospital-name"
